src/sentry/monitors/endpoints/organization_monitor_index.py

```diff
@@ -106,9 +106,11 @@ def get(self, request: Request, organization: Organization) -> Response:
             if request.GET.get("includeNew"):
                 queryset = queryset.filter(
                     Q(monitorenvironment__environment__in=environments) | Q(monitorenvironment=None)
-                )
+                ).distinct()
```

Comment:
should we leave a small comment here as to why we need this?

Reply:
can add it later